.github/workflows/ci-tag-deploy.yml

```diff
@@ -0,0 +1,87 @@
+name: Live Deploy
+
+on:
+  push:
+    # Pattern matched against refs/tags
+    tags:        
+      - '*'
+
+permissions:
+  id-token: write
+  contents: read
+
+jobs:
+  live-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Kick off Terraform deploy in sysops/
+      id: sysops-deploy
+      run: |
+        curl \
+          -X POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer ${{ secrets.SYSOPS_RW_GITHUB_TOKEN }}" \
+          -H "X-GitHub-Api-Version: 2022-11-28" \
+          https://api.github.com/repos/meedan/sysops/actions/workflows/deploy_${{ github.event.repository.name }}.yml/dispatches \
```

Comment:
Does this trigger script to look at the tagged repository and redeploy all of the associated services?   Are we assuming that the workflow is "merge to main, run in QA for a while, then tag to trigger live release?" i.e. the images to be deployed are already build and pushed to ECR?